pom.xml

```diff
@@ -15,7 +15,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.curityVersion>4.4.0</project.curityVersion>
         <project.slf4jVersion>1.7.25</project.slf4jVersion>
-        <kotlin.version>1.2.50</kotlin.version>
+        <kotlin.version>1.3.72</kotlin.version>
```

Comment:
The 5.0 docs say that the version that Curity provides is `1.3.61`. Are those out of date or is this version newer than it ought to be?

Reply:
True, changing that